src/components/Inventory/Inventory.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import InventoryItem from "../InventoryItem/InventoryItem";
+export default function Inventory({ inventory, strength }) {
+  let totalWeight = 0;
+  inventory.forEach((item) => {
+    totalWeight += Object.values(item)[0].weight * item.amount;
+  });
+  return (
+    <table>
```

Comment:
Try not to use the tag table for style, just when you are rendering data as a table